build.gradle

```diff
@@ -1908,6 +1908,7 @@ project(':tools') {
     testImplementation project(':connect:api')
     testImplementation project(':connect:runtime')
     testImplementation project(':connect:runtime').sourceSets.test.output
+    testImplementation project(':storage:api').sourceSets.main.output
```

Comment:
@nizhikov 
This gradle change is breaking `trunk` builds for me locally, should this be `:storage:storage-api` instead?

Reply:
I see builds fail on jenkins similarly -
https://lists.apache.org/thread/qnscvw8nb3dscrkm43r7g5224067jzc6

Reply:
Hello. 

Please, take a look at jenkins build of trunk after this merged - https://ci-builds.apache.org/blue/organizations/jenkins/Kafka%2Fkafka/detail/trunk/2248/pipeline

Looks like it builds successfully.
Can you, please, clarify, how you trying to build trunk and what exactly fails?

Reply:
It seems like 7553d3f562f3af6c7f9b062b9220bcad80b00478 breaks compilation.

Reply:
I will provide fix in the nearest time.

Reply:
Fix - #14475

Reply:
@msn-tldr Fix merge e90f82b62a94dc556e231f033e22030c06d60232

Reply:
@nizhikov i can see its fixed in trunk, thanks.